### ISSUE TITLE
Mathml font issue

### ISSUE THREAD

**Issue by Intelligent2013** [metanorma/mn-native-pdf#255]
Specified font for mathml doesn't any affect.

Example xslt. Just duplicate math equation in two fonts - 'Cambria Math' and 'STIX2Math':
```xml
	<xsl:template match="mathml:math" priority="2">
		<fo:block><fo:inline padding-right="5mm">Cambria Math</fo:inline>
			<fo:inline font-family="Cambria Math">
				<xsl:variable name="mathml">
					<xsl:apply-templates select="." mode="mathml"/>
				</xsl:variable>
				<fo:instream-foreign-object fox:alt-text="Math">
					<!-- <xsl:copy-of select="."/> -->
					<xsl:copy-of select="xalan:nodeset($mathml)"/>
				</fo:instream-foreign-object>
			</fo:inline>
		</fo:block>
		<fo:block><fo:inline padding-right="5mm">STIX2Math</fo:inline>
			<fo:inline font-family="STIX2Math">
				<xsl:variable name="mathml">
					<xsl:apply-templates select="." mode="mathml"/>
				</xsl:variable>
				<fo:instream-foreign-object fox:alt-text="Math">
					<!-- <xsl:copy-of select="."/> -->
					<xsl:copy-of select="xalan:nodeset($mathml)"/>
				</fo:instream-foreign-object>
			</fo:inline>
		</fo:block>
	</xsl:template>
```

The resulted images are identical:
![изображение](https://user-images.githubusercontent.com/5559926/94307691-eec36880-ff7d-11ea-86b2-27840126b436.png)

**Comment by Intelligent2013** [comment]
Comparison with 'Times New Roman' and 'Arial'. There is a difference:
![изображение](https://user-images.githubusercontent.com/5559926/94308842-ce94a900-ff7f-11ea-9ffa-1f1f808b6ea7.png)

**Comment by Intelligent2013** [comment]
One more example:
![изображение](https://user-images.githubusercontent.com/5559926/94339570-80c18480-0003-11eb-9a83-34e83b59382f.png)

Each row in the table displays with own font. In the right column equation shows as simple text with font. In middle column - generated by jEuclid.
Apache FOP config has all fonts, except 'TestMathFont'.
Note, that both fonts 'Times New Roman' and 'Times New Roman2' determined in FOP config, BUT in middle column there is a differences:
- image for 'Times New Roman' looks similar to text in right column
- image for 'Times New Roman2' looks similar to image in font 'TestMathFont'.

May be font-family for math images should be exactly same as determined internally in font file...

**Comment by Intelligent2013** [comment]
New results:
- jEuclid uses real font name from font file (not file name), but Apache FOP can process any font-triplet name determined on config. Tasks:
   - in FOP config `<font-triplet name="STIX2Math"` should replaced by `<font-triplet name="STIX Two Math"`, 
   -  in all xslt `STIX2Math' -> `STIX Two Math`.

See row 'Font STIX Two Math'. In middle column (jEuclid) `beta` glyph looks identical to `beta` in right column (FOP).

![изображение](https://user-images.githubusercontent.com/5559926/94341808-24fff700-0015-11eb-9677-6936cb6de323.png)

- `Cambria Math` was temporarely removed from FOP config (as you can see # at 1st row, last column), but jEuclid shows glyphs in any case. Looks like there an internal glyphs source in jEuclid, Need to be determined...

- in 4th row `Font STIX Two Math` doesn't contain some glyphs (see # in last column), but jEuclid shows them. Looks like jEuclid can put the glyphs from different fonts/source.

**Comment by Intelligent2013** [comment]
I've investigated jEuclid sources and how font selection works:
- load information about all fonts from AWT (Abstract Window Toolkit) `GraphicsEnvironment.getLocalGraphicsEnvironment().getAvailableFontFamilyNames();`
I.e. all fonts, available in system for programs.
- load information about all fonts from classpath with mimetype `"application/x-font"`
- for each char in math, if outer fo element for mathml (`fo:inline`) has an attribute `font-family` (like STIX Two Math), then jEuclid try to find it in AWT fonts, then in classpath fonts, and if it is not found then uses first font which has a glyph (code point in terms of jEuclid) for current char. I.e. even we set font in Apache FOP, jEuclid can use another font. On my previous screenshot font 'STIX Two Math' display ok, because it presents in Windows/Fonts folder. As soos as I removed it, then it displays in 'any' font, as 'TestFontMath'

New Task: add all fonts from `font-path` in mn2pdf in classpath, or try to make visible in via AWT system.
- for each glyph
